### PR TITLE
Updating with new logo & link to TT.io

### PR DESCRIPTION
**Pull request by vtimd**
New logo, uploaded to repo and added to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Terratag by env0 [![ci](https://github.com/env0/terratag/workflows/ci/badge.svg)](https://github.com/env0/terratag/actions?query=workflow%3Aci+branch%3Amaster) [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fenv0%2Fterratag.svg?type=small)](https://app.fossa.com/projects/git%2Bgithub.com%2Fenv0%2Fterratag?ref=badge_small)
+# [<img src="ttlogo.png" width="300">](https://terratag.io) [![ci](https://github.com/env0/terratag/workflows/ci/badge.svg)](https://github.com/env0/terratag/actions?query=workflow%3Aci+branch%3Amaster) [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fenv0%2Fterratag.svg?type=small)](https://app.fossa.com/projects/git%2Bgithub.com%2Fenv0%2Fterratag?ref=badge_small)
```
Comment (avnerenv0) [inline README.md:1]: I guess we wanna have https://terratag.io up and running before we merge this?

Comment (vtimd) [reply README.md:1]: Omry should be setting the DNS records soon. Probably not a huge deal, but it can wait until the site is live.

Comment (vtimd) [reply README.md:1]: Terratag.io is live, we're good to merge.

Comment (omry-hay) [inline README.md:1]:
```suggestion
[![ci](https://github.com/env0/terratag/workflows/ci/badge.svg)](https://github.com/env0/terratag/actions?query=workflow%3Aci+branch%3Amaster) [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fenv0%2Fterratag.svg?type=small)](https://app.fossa.com/projects/git%2Bgithub.com%2Fenv0%2Fterratag?ref=badge_small)

# [<img src="ttlogo.png" width="300">](https://terratag.io) 
```

**Review by omry-hay — CHANGES_REQUESTED** [review]
It looks quite bad that the CI badges are next to the logo, so let's make them on top before the logo please.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# Terratag by env0 [![ci](https://github.com/env0/terratag/workflows/ci/badge.svg)](https://github.com/env0/terratag/actions?query=workflow%3Aci+branch%3Amaster) [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fenv0%2Fterratag.svg?type=small)](https://app.fossa.com/projects/git%2Bgithub.com%2Fenv0%2Fterratag?ref=badge_small)
+[![ci](https://github.com/env0/terratag/workflows/ci/badge.svg)](https://github.com/env0/terratag/actions?query=workflow%3Aci+branch%3Amaster) [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fenv0%2Fterratag.svg?type=small)](https://app.fossa.com/projects/git%2Bgithub.com%2Fenv0%2Fterratag?ref=badge_small)
+
+# [<img src="ttlogo.png" width="300">](https://terratag.io)
```
Comment (avnerenv0) [inline README.md:3]: Sorry to be a stickler but can we have the logo above the badges ? that's usually the format.
Also can we add an `alt` to the `img` ?

Comment (vtimd) [reply README.md:3]: No worries, I’ll take care of this.

Comment (vtimd) [reply README.md]: Moved the logo, and added alt text.

**Review by avnerenv0 — APPROVED** [review]
👍

**Pull request merged**